data-prepper-plugins/opensearch/build.gradle

```diff
@@ -55,7 +55,7 @@ dependencies {
     implementation 'software.amazon.awssdk:utils:2.17.15'
     implementation 'software.amazon.awssdk:sts:2.17.15'
     implementation 'software.amazon.awssdk:url-connection-client:2.17.15'
-    implementation "io.micrometer:micrometer-core:1.7.2"
+    implementation 'io.micrometer:micrometer-core:1.7.3'
```

Comment:
Will it lead to build issue without the hardcoded version number here?

Reply:
We will have to manually maintain these. I'm not sure why, but something with the OpenSearch Gradle plugins that we use is causing BOMs to not work as expected. So this project is going to operate as an exception by having to specify its dependencies. This may be something we can resolve with more effort.